### PR TITLE
Add package typhos

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc @hhslepicka

### CODE REVIEW

**Comment by klauer** [comment]
Dependencies in need of updating/addition to conda-forge:
- [ ] ophyd 1.4 release (expected later today)
- [ ] timechart to be added to conda-forge (@hhslepicka will submit)

**Comment by hhslepicka** [comment]
I accept to be a maintainer of this package.

**Comment by hhslepicka** [comment]
Ophyd conda-forge feedstock PR is now open: https://github.com/conda-forge/ophyd-feedstock/pull/16

**Comment by hhslepicka** [comment]
Timechart conda-forge recipe PR is now open: https://github.com/conda-forge/staged-recipes/pull/11065

**Comment by hhslepicka** [comment]
@conda-forge-admin, please restart ci

**Comment by hhslepicka** [comment]
@conda-forge-admin, please restart ci

**Comment by klauer** [comment]
@conda-forge-admin, please restart ci

I can find the newly-available-on-conda-forge `timechart` now, let's hope CI can find it:
```
$ conda search timechart
Loading channels: done
# Name                       Version           Build  Channel
timechart                      1.2.3    pyh8c360ce_0  conda-forg
```

**Comment by klauer** [comment]
Last step, I hope: need a new pydm tag, @hhslepicka.

Imports failing with: `ModuleNotFoundError: No module named 'pydm.exception'`

**Comment by hhslepicka** [comment]
@klauer, tag created... soon to be available at conda-forge

**Comment by klauer** [comment]
@conda-forge-admin, please restart ci

**Comment by klauer** [comment]
@conda-forge/help-python this is finally good to go 😃

File: recipes/typhos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  url: https://github.com/pcdshub/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cdcc30a6e1a7eeb72f2a0b9db711dd28cbfc073e3c39e9ac2b7bfaa7fbb31821
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - typhos = typhos.cli:main
+    - typhon = typhos.cli:main
```
Comment (npavlovikj) [inline recipes/typhos/meta.yaml]: What is the difference between `typhos` and `typhon` ?

File: recipes/typhos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+test:
+  imports:
+    - typhos
+    - typhos.cli
+
+  commands:
+    - typhos --help
+
+  requires:
```
Comment (npavlovikj) [inline recipes/typhos/meta.yaml]: Are these required to test the import and the command?

File: recipes/typhos/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    - pytest-qt
+    - pytest-timeout
+
+about:
+  home: https://github.com/pcdshub/typhos
+  summary: 'Automatic-yet-customizable Graphical User Interface Generation for Ophyd Devices'
+  license: SLAC Open
+  license_family: OTHER
+  license_file: LICENSE.txt
+
```
Comment (npavlovikj) [inline recipes/typhos/meta.yaml]: You can also add:
```
doc_url: https://pcdshub.github.io/typhos
dev_url: https://github.com/pcdshub/typhos
```

File: recipes/typhos/meta.yaml (hunk 1 above)
Comment (npavlovikj) [reply recipes/typhos/meta.yaml]: Ah, I see the package for renamed. Is that why?

File: recipes/typhos/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    - timechart
+
+test:
+  imports:
+    - typhos
+    - typhos.cli
+
+  commands:
+    - typhos --help
+
```
Comment (npavlovikj) [inline recipes/typhos/meta.yaml]: I would also add `typhon  --here` here.

File: recipes/typhos/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    - qdarkstyle
+    - qtawesome
+    - qtconsole
+    - timechart
+
+test:
+  imports:
+    - typhos
+    - typhos.cli
+
```
Comment (npavlovikj) [inline recipes/typhos/meta.yaml]: This is a stylish comment, but please remove the empty lines within a Section.

File: recipes/typhos/meta.yaml (hunk 1 above)
Comment (klauer) [reply recipes/typhos/meta.yaml]: Yes, this is something that will be removed in the next version, but remains here for back-compatibility for now.

File: recipes/typhos/meta.yaml (hunk 2 above)
Comment (klauer) [reply recipes/typhos/meta.yaml]: It was a placeholder for running the actual tests on conda-forge once we have our own feedstock. I'll remove them for now.

Comment (klauer) [reply recipes/typhos/meta.yaml]: Addressed in 9a96456a1f33c838472f0ce77b0829aba67f85f6

File: recipes/typhos/meta.yaml (hunk 3 above)
Comment (klauer) [reply recipes/typhos/meta.yaml]: Addressed in 9a96456a1f33c838472f0ce77b0829aba67f85f6

File: recipes/typhos/meta.yaml (hunk 4 above)
Comment (klauer) [reply recipes/typhos/meta.yaml]: Addressed in 9a96456a1f33c838472f0ce77b0829aba67f85f6

File: recipes/typhos/meta.yaml (hunk 5 above)
Comment (klauer) [reply recipes/typhos/meta.yaml]: Addressed in 9a96456a1f33c838472f0ce77b0829aba67f85f6 - I think I got them all.

**Comment by npavlovikj** [comment]
The recipe looks good @klauer , thank you for addressing the comments.

**Comment by klauer** [comment]
Thanks, @npavlovikj!

**Pull request merged**